pages/common/montage.md

```diff
@@ -0,0 +1,24 @@
+# montage
+
+> Imagemagick image montage tool.
+> Tiles images into a customisable grid.
+
+- Tile images into a grid, automatically resizing images larger than the grid size:
+
+`montage {{image1.png}} {{image2.jpg}} {{imageN.png}} montage.jpg`
+
+- Tile images into a grid, automatically calculating the grid size from the largest image:
+
+`montage {{image1.png}} {{image2.jpg}} {{imageN.png}} -geometry +0+0 montage.jpg`
+
+- Set the grid size and resize images to fit before tiling:
+
+`montage {{image1.png}} {{image2.jpg}} {{imageN.png}} -geometry 640x480+0+0 montage.jpg`
+
+- Limit size of grid, causing input images to overflow into multiple output images:
```

Comment:
Ok, if I'm understanding this correctly, so far you seem to have used "size" to mean the dimensions of images or the grid; but now you're talking about the number of rows and columns, right? Or am I reading this wrong? Either way, I believe we should remove this ambiguity before merging. I'm sorry not to have caught this earlier :/